S2VX.Game/Editor/ToolState/SelectToolState.cs

```diff
@@ -67,7 +135,7 @@ public class SelectToolState : S2VXToolState {
 
             foreach (var noteAndTime in SelectedNoteToTime) {
                 var note = noteAndTime.Key;
-                if (MouseIsOnNote(mousePos, Editor.NotesTimeline.NoteToTimelineNote[note])) {
+                if (Editor.NotesTimeline.NoteToTimelineNote.ContainsKey(note) && MouseIsOnTimelineNote(mousePos, Editor.NotesTimeline.NoteToTimelineNote[note])) {
```

Comment:
This line is >150 chars

Reply:
I think limiting line length is fine, but shouldn't be fixed through review. I made issue #105 to add this package https://github.com/menees/Analyzers to check for line length.